spec/ddtrace/meta_header_spec.rb

```diff
@@ -1,101 +0,0 @@
-require 'spec_helper'
```

Comment:
Removed this integration test since it overlaps with the HTTP adapter integration tests, which are responsible for tagging the request with the correct HTTP headers. Hence, this is considered obsolete.